framework/core/render_pass.cpp

```diff
@@ -438,7 +492,7 @@ void RenderPass::create_renderpass(const std::vector<Attachment> &attachments, c
 		color_output_count.push_back(to_u32(color_attachments[i].size()));
 	}
 
-	const auto &subpass_dependencies = get_subpass_dependencies<T_SubpassDependency>(subpass_count);
+	const auto &subpass_dependencies = get_subpass_dependencies<T_SubpassDependency>(subpass_count, is_depth_a_dependency<T_SubpassDescription, T_AttachmentDescription>(subpass_descriptions, attachment_descriptions));
```

Comment:
I think, there's no need to explicitly list the template arguments for `is_depth_a_dependency`. The compiler should be able to identify them anyways.

Reply:
Done